app/pages/VatRatesFromNiPage.scala

```diff
@@ -31,10 +31,10 @@ case class VatRatesFromNiPage(index: Index) extends QuestionPage[List[VatRate]]
   override def toString: String = PageConstants.vatRates
 
   override def navigateInNormalMode(answers: UserAnswers): Call =
-    routes.NetValueOfSalesFromNiController.onPageLoad(NormalMode, answers.period, index, Index(0))
+    routes.SalesAtVatRateFromNiController.onPageLoad(NormalMode, answers.period, index, Index(0))
```

Comment:
Cant remember if we were coming back to this later or not but should this Index be dynamic?

Reply:
same as below

Reply:
No, because if it changes, it should clear all the data, therefore will always want to start at 0. To my understanding, when we change and add an additional vat rate, it will clear all previous and ask user to enter all of them (because of the misaligned json issue)